src/utils/ipc.ts

```diff
@@ -0,0 +1,31 @@
+import {Server, connect} from 'net';
+import {ipc_port} from '../config.json';
+import {Collection} from 'discord.js';
+import * as log from './log';
+
+const requestHandlers: Collection<string, () => void> = new Collection();
+
+export function on( event: string, handler: () => void) {
+	log.writeToLog( undefined, `A listener for the event ${event} was registered` );
```

Comment:
This will just clog the log because it'll print this every time the bot spins up